### PR TITLE
Content/co tweaks

### PR DESCRIPTION
**Pull request by wodenx**
Various content tweaks on home page
Add a desc ription to styleguide page

### CODE REVIEW

File: sites/bodiless-demo/src/data/pages/bottom-content$48eb0b70-dad7-11ec-b70a-a1c4c7d90965$description.json [hunk 1]
```diff
@@ -3,7 +3,23 @@
     "type": "paragraph",
     "children": [
       {
-        "text": "For larger organization, content reuse can be a significant efficiency. A centralized content library can be shared between many sites which can inherit directly or customize."
+        "text": "For some sites, content reuse can be a significant efficiency, but you don't need a costl CMS application to achieve it.  In Bodiless, a centralized content library can be shared across your site.  Components with content can be added to the library and then reused on multiple pages."
```
Comment (dewen) [inline sites/bodiless-demo/src/data/pages/bottom-content$48eb0b70-dad7-11ec-b70a-a1c4c7d90965$description.json]: costl => costly

Comment (hvanyo) [inline sites/bodiless-demo/src/data/pages/bottom-content$48eb0b70-dad7-11ec-b70a-a1c4c7d90965$description.json]:
```suggestion
        "text": "For some sites, content reuse can be a significant efficiency, but you don't need a costly CMS application to achieve it.  In Bodiless, a centralized content library can be shared across your site.  Components with content can be added to the library and then reused on multiple pages."
```

File: sites/bodiless-demo/src/data/pages/design-system.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      {
+        "text": "Each component can be used out of the box with minimal theming, or extended and customized to match almost any design.  All are fully editable, and implemented with best-in-class accessibility, SEO and performance in mind."
+      }
+    ]
+  },
+  {
+    "type": "paragraph",
+    "children": [
+      {
+        "text": "By the way - this page is an exmple of the built-in Bodiless content listing component.  Click the "
```
Comment (hvanyo) [inline sites/bodiless-demo/src/data/pages/design-system.json]:
```suggestion
        "text": "By the way - this page is an example of the built-in Bodiless content listing component.  Click the "
```

File: sites/bodiless-demo/src/data/pages/features/main-content$a16a5bd0-ecc4-11ec-8f8a-8f73f554bfda.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
     "type": "paragraph",
     "children": [
       {
-        "text": "The features are compared to other non-cms JAMStack."
+        "text": "These days, \"no-cms\" site buildng frameworks are becoming more and more common.  Bodiless was one of the fist, and has been under intensive development for nearly four years.  As a result, it's packed with features which are hard to find in similar systems."
```
Comment (hvanyo) [inline sites/bodiless-demo/src/data/pages/features/main-content$a16a5bd0-ecc4-11ec-8f8a-8f73f554bfda.json]:
```suggestion
        "text": "These days, \"no-cms\" site buildng frameworks are becoming more and more common.  Bodiless was one of the first, and has been under intensive development for nearly four years.  As a result, it's packed with features which are hard to find in similar systems."
```

File: sites/bodiless-demo/src/data/pages/features/main-content$a16a5bd0-ecc4-11ec-8f8a-8f73f554bfda.json [hunk 4]
```diff
@@ -11,7 +11,7 @@
     "type": "paragraph",
     "children": [
       {
-        "text": "The features are compared to other non-cms JAMStack."
+        "text": "These days, \"no-cms\" site buildng frameworks are becoming more and more common.  Bodiless was one of the first, and has been under intensive development for nearly four years.  As a result, it's packed with features which are hard to find in similar systems."
```
Comment (hvanyo) [inline sites/bodiless-demo/src/data/pages/features/main-content$a16a5bd0-ecc4-11ec-8f8a-8f73f554bfda.json]:
```suggestion
        "text": "These days, \"no-cms\" site building frameworks are becoming more and more common.  Bodiless was one of the first, and has been under intensive development for nearly four years.  As a result, it's packed with features which are hard to find in similar systems."
```

**Review by hvanyo — APPROVED** [review]

**Pull request merged**